MonMaraicher/Features/Map/MapView.swift

```diff
@@ -13,22 +13,24 @@ struct MapView: View {
     @StateObject private var viewModel = MapViewModel(farmerService: FarmerService())
 
     var body: some View {
-        Map(position: $viewModel.mapCameraPosition, selection: $viewModel.selectedFarmer) {
+        Map(position: $viewModel.mapCameraPosition, selection: $viewModel.selectedAddress) {
 
             ForEach(viewModel.allFarmers, id: \.id) { farmer in
-                Marker(farmer.title,
-                       systemImage: farmer.systemImageName,
-                       coordinate: CLLocationCoordinate2D(latitude: farmer.location.latitude, longitude: farmer.location.longitude))
-                .tag(farmer)
-                .tint(.orange)
+                ForEach(farmer.adressesOperateurs, id: \.id) { location in
+                    Marker(farmer.raisonSociale.capitalized, systemImage: "laurel.leading", coordinate: .init(latitude: location.lat, longitude: location.long))
```

Comment:
`raisonSociale` 😅

Reply:
Don't you know "raisonSociale" in English? ahah